index.js

```diff
@@ -1,20 +1,16 @@
 module.exports = {
   extends: ['standard', 'standard-react'],
   rules: {
-    /*
-      Standard overrides
-    */
-    semi: ['error', 'always'], // churn-expensive to enforce
-    'space-before-function-paren': ['error', 'never'],
-
     /*
       Nava-specific rules
     */
-    'linebreak-style': ['error', 'unix'],
-    'sort-vars': ['error', {ignoreCase: true}],
     'no-alert': 'error',
     'no-var': 'error',
     'prefer-const': 'error',
-    'prefer-arrow-callback': 'error'
+    'prefer-arrow-callback': 'error',
```

Comment:
I guess I'm still not totally sure I see the boundary between style rules + correctness rules -- I can see an argument for `prefer-arrow-callback` being a style rule and not a correctness rule. The two examples given in the [description of the rule](https://eslint.org/docs/rules/prefer-arrow-callback#rule-details) both seem to me like things that wouldn't change any behavior but just look nicer 🤷‍♂️ 

The other rules make sense to me because they're preventing you from doing something dumb, but `prefer-arrow-callback` seems like more of a stylistic thing to me? Unless I'm missing something?

Reply:
Yeah I'd say `prefer-arrow-callback` is not a correctness rule, but it does encourage the use of new es6 features that are terse or give more signal (like const/let vs var), and it's also not a controversial rule. I guess the main thing I wanted to change was to avoid purely stylistic rules; like spacing, semis, etc.